sormas-e2e-tests/src/test/resources/features/sanity/web/DocumentTemplate.feature

```diff
@@ -0,0 +1,58 @@
+@UI @Sanity @DocumentTemplates
+Feature: Create user
+
+  @issue=SORDEV-5497
+  Scenario: Upload Case Document Template
+    Given I log in as a Admin User
+    And I click on the Configuration button from navbar
+    And I navigate to document templates tab
+    When I click on the UPLOAD TEMPLATE button from Document Templates Case
+    And I pick the case document template file
+    And I click on the UPLOAD TEMPLATE button from the popup
+    And I confirm the overwrite popup
+    Then I check that an upload success notification appears
+
+  @issue=SORDEV-5497
+  Scenario: Upload Contact Document Template
+    Given I log in as a Admin User
+    And I click on the Configuration button from navbar
+    And I navigate to document templates tab
+    When I click on the UPLOAD TEMPLATE button from Document Templates Contact
+    And I pick the contact document template file
+    And I click on the UPLOAD TEMPLATE button from the popup
+    And I confirm the overwrite popup
+    Then I check that an upload success notification appears
+
+  @issue=SORDEV-5497
+  Scenario: Upload Event Participant Document Template
+    Given I log in as a Admin User
+    And I click on the Configuration button from navbar
+    And I navigate to document templates tab
+    When I click on the UPLOAD TEMPLATE button from Document Templates Event Participant
+    And I pick the event participant document template file
+    And I click on the UPLOAD TEMPLATE button from the popup
+    And I confirm the overwrite popup
+    Then I check that an upload success notification appears
+
+  @issue=SORDEV-5497
+  Scenario: Upload Travel Entry Document Template
```

Comment:
For some reason, testing this document upload was not mentioned in the manual test scenario 
(perhaps it was written before this functionality was added?)
I figured that it wouldn'd hurt to add it though, so that all the available template uploads are tested.

Reply:
I'm happy with the approach :D can you confirm with Kay if we can create also a test case for this if there's nothing?

Reply:
Kay approves.

> Hi Jedrzej, yes that would be great, I think the old test might just not have been updated after the document generation feature was added to travel entries